sdk/azcore/policy_retry.go

```diff
@@ -21,6 +21,10 @@ const (
 	RetryPolicyFixed RetryPolicy = 1
 )
 
+const (
+	defaultMaxRetries = 4
```

Comment:
Is it MaxRetries or max tries? I used MaxTries in the Storage libraries because 0 tries makes no sense and 0 is the default value so I could set it to default max tries. If it's max retries then 0 DOES make sense; 1 try and no retries. So then 0 can't be a sentinel value to change it to default max tries.

Reply:
You are right it should be `defaultMaxTries`, will fix in next round of clean-up.